pkg/result/filter.go

```diff
@@ -67,8 +70,19 @@ func FilterResult(ctx context.Context, result *types.Result, ignoreConf IgnoreCo
 	filterLicenses(result, severities, opt.IgnoreLicenses, ignoreConf)
 
 	if opt.PolicyFile != "" {
-		if err := applyPolicy(ctx, result, opt.PolicyFile); err != nil {
-			return xerrors.Errorf("failed to apply the policy: %w", err)
+		log.Logger.Debugf("Filtering result with ignore policies, type: %s, path: %s", result.Type, result.Target)
```

Comment:
nit:
I'm not sure we need to write new log messages.
This can be noisy because we will show this message for each `Result` and the message for each policy file for each `Result`.

Reply:
Thank you for the review.
Sure. Removed the logging.

Initially I was thinking that was fine, as it logged a line per each result item. E.G.
```
2024-03-19T16:35:09.938+0200	DEBUG	Detecting library vulnerabilities, type: pip, path: requirements.txt
2024-03-19T16:35:09.942+0200	DEBUG	Detecting library vulnerabilities, type: npm, path: package-lock.json
```

Reply:
this is log message only for vulnerabilities.
But in this case we will show logs for vulns, misconfigs, secrets, licenses. 
There may be too many of them.

We added https://github.com/aquasecurity/trivy/blob/8ec3938e01a93855503e3400eae9831abbb5de4a/pkg/types/finding.go#L31
To show filtered elements.